### DISCUSSION TITLE
Passing a variable to a snippet

### DISCUSSION THREAD

**Discussion by esamecar** [ttscoff/bunch-docs#174] [Q&A]
Hi,

sorry, might be a RTFM thing but I cannot figure out why

```
# diverse Variabeln für das projekt.snippet
pfad = "/Users/justsomeone/"
dfx = "Forschung"
timerytimer = "Forschung"
git = true

< Projekte/projekt.snippet
```

does work as expected - I can use the four variables in the snippet, but

```
< Projekte/projekt.snippet
- pfad = "/Users/justsomeone/"
- dfx = "Forschung"
- timerytimer = "Forschung"
- git = true
```

does not work. Do I have to define variables before passing them?

Thanks for clarification!

Kind regards

Marco

**Comment by ttscoff** [comment]
Not a RTFM thing, your example should work. Will have to do some testing 
next time I have a chance to dig in. I'll have followup questions when I 
get to it, will keep you posted.

-Brett
